controllers/module_reconciler.go

```diff
@@ -347,9 +348,13 @@ func (mrh *moduleReconcilerHelper) handleSigning(ctx context.Context, mld *api.M
 func (mrh *moduleReconcilerHelper) handleDriverContainer(ctx context.Context,
 	mld *api.ModuleLoaderData) error {
 
-	hashValue, err := hashstructure.Hash(hashData{KernelVersion: mld.KernelVersion}, hashstructure.FormatV2, nil)
+	dsNameData := hashData{
+		KernelVersion: mld.KernelVersion,
+		ModuleVersion: mld.ModuleVersion,
+	}
+	hashValue, err := hashstructure.Hash(dsNameData, hashstructure.FormatV2, nil)
```

Comment:
Why don't we garbage collect `moduleLoader`'s daemonsets as well?

Reply:
This code does not deal with Garbage collection, only with creation of the Daemonsets